pom.xml

```diff
@@ -207,6 +207,17 @@
             <version>5.2.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-resolver-dns-native-macos</artifactId>
+            <classifier>osx-aarch_64</classifier>
+            <version>4.1.108.Final</version>
+        </dependency>
```

Comment:
I needed this to run it on my M2 Mac

Reply:
This dependency should be declared with `<scope>runtime</scope>`